docs/en/observability/quickstarts/collect-data-with-aws-firehose.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[collect-data-with-aws-firehose]]
+= Quickstart: Collect data with AWS Firehose
+
+preview::[]
+
+In this quickstart guide, you'll learn how to use AWS Firehose to send logs and metrics to Elastic.
+
+The AWS Firehose streams are created using a CloudFormation template, which can collect all available CloudWatch logs and metrics for your AWS account.
+
+This approach requires minimal configuration as the CloudFormation template creates a Firehose stream, enables CloudWatch metrics collection across all namespaces, and sets up an account-level subscription filter for CloudWatch log groups to send logs to Elastic via Firehose.
+Optionally, you can use an AWS CLI command or upload the template to the AWS CloudFormation portal to customize parameter values.
+
+IMPORTANT: Some AWS services need additional manual configuration to properly ingest logs and metrics. For more information, check the 
+link:https://www.elastic.co/docs/current/integrations/aws[AWS integration] documentation.
+
+Data collection with AWS Firehose is supported on ESS deployments in AWS, Azure and GCP. 
+
+[discrete]
+== Prerequisites
```

Comment:
@alaudazzi could we please also include a note that be default CloudFormation stack will be created in the AWS region selected for the user's account, and this can be changed either in AWS Console UI or by adding a `--region` parameter to the AWS CLI command while creating the stack.